### PR TITLE
[MRG+1] DOC MEP12 - converted animation to SG/MEP12 compatible

### PR DESCRIPTION
**Pull request by NelleV**
Minor modification to have part of the animation section Sphinx-gallery compatible.

I'm pretty sure there is a problem in the Baye's update example, considering it does not converge to the theoretical value the code mentions…

Supersedes: #6486

See #7206

### CODE REVIEW

File: examples/animation/animate_decay.py [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
+
+    Generates a decaying sine wave.
+    """
+    for count in range(1000):
         t += 0.1
         yield t, np.sin(2*np.pi*t) * np.exp(-t/10.)
 
 
 def init():
+    """Create a line
```
Comment (QuLogic) [inline examples/animation/animate_decay.py]: This is not quite correct. It doesn't create the line, it updates the line's data with the initial value, which is maybe a bit odd, I guess.

File: examples/animation/animate_decay.py [hunk 2]
```diff
@@ -1,17 +1,31 @@
+"""
+=====
+Decay
```
Comment (QuLogic) [inline examples/animation/animate_decay.py]: Probably needs another word, "Decay animation", or something.

File: examples/animation/basic_example.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
+"""
+=========================
+Simple animation examples
+=========================
+
+This example contains two animations. The first is a random walk plot. the
```
Comment (QuLogic) [inline examples/animation/basic_example.py]: the -> The

File: examples/animation/strip_chart_demo.py [hunk 4]
```diff
@@ -1,4 +1,8 @@
 """
+============
+Oscilloscope
+============
+
 Emulate an oscilloscope.  Requires the animation API introduced in
 matplotlib 1.0 SVN.
```
Comment (QuLogic) [inline examples/animation/strip_chart_demo.py]: Do we need this information any more?

**Comment by tacaswell** [comment]
Thanks

Backported to v2.x as 66941ec

**Pull request merged**